Homework-7/Enum/Country.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Homework_7.Enum
+{
+    public class Country
+    {
+        public enum CountryType
+        {
+            USA,
+            Canada,
+            UK,
+            Australia
+        }
+    }
```

Comment:
It isn't necessary to create a class around the enum.. you can directly create enum.